pkg/obfuscate/sql.go

```diff
@@ -181,7 +181,7 @@ func (f *replaceFilter) Filter(token, lastToken TokenKind, buffer []byte) (token
 	switch lastToken {
 	case Savepoint:
 		return markFilteredGroupable(token), questionMark, nil
-	case '=':
+	case '=', LE, GE, NE:
 		switch token {
 		case DoubleQuotedString:
 			// double-quoted strings after assignments are eligible for obfuscation
```

Comment:
Out of curiosity, do we know why / should this be *only* DoubleQuotedString tokens?

Reply:
If I read line 192 correctly, all other types of strings are already substituted by default, only double quoted strings are not - because depending on ANSI_QUOTES setting in mysql it may also be a column name.
But apparently we decided here to err on the side of only obfuscating the right-hand argument of the equals sign for it.